platform/src/main/java/org/stellar/anchor/platform/AnchorPlatformServer.java

```diff
@@ -87,4 +90,9 @@ public EventPublishService eventService(
             String.format("Invalid event publisher: %s", eventConfig.getPublisherType()));
     }
   }
+
+  @Bean
+  public MetricEmitterService metricService(JdbcSep31TransactionStore sep31TransactionStore) {
```

Comment:
This ties the metric emitter service to the specific JdbcSep31TransactionStore. If we switch the transaction store to database other than JDBC, such as Mongo, this will fail. 

IMO, the metrics should be considered separately from the analytics. The number of transactions matching certain status should belong to the analytics while the number of the transactions should belong to metrics. 

I think querying the database with status filter for Prometheus scraping may hinder the scalability. Can we not query the database at this moment? We may discuss this if you have other thoughts.